setup.py

```diff
@@ -137,6 +137,11 @@
     # Download Model Files #
     ########################
     "wget>=3.2,<4.0",
+
+    ##################
+    # Text to Speech #
+    ##################
+    "pt-pyfestival",
```

Comment:
```suggestion
    "pyfestival",
```

Reply:
This will not install correctly for people installing via Python on RPi for now, but we want tackle that later on.